modules/gapi/src/streaming/onevpl/accelerators/surface/dx11_frame_adapter.hpp

```diff
@@ -37,6 +37,9 @@ class VPLMediaFrameDX11Adapter final: public BaseFrameAdapter,
     GAPI_EXPORTS ~VPLMediaFrameDX11Adapter();
     MediaFrame::View access(MediaFrame::Access) override;
 
+    // FIXME: Consider a better solution since this approach
+    // is not easily extendable for other adapters (oclcore.cpp)
+    cv::util::any getHandle();
```

Comment:
visitor/acceptor

Reply:
I think returning concreate type would be enough

Reply:
Type is fixed. `Visitor/acceptor option are considered for future redesign if needed